---
title: "About"
permalink: /about/
toc: true
toc: true
toc_label: "Jump to:"
toc_icon: "fast-forward"
---

I have 10 years of experience modeling data in the context of molecular
and cellular biology in order to gain insights and make better
experimental decisions. I am interested in opportunities to apply my
skills to relevant problems in and outside biology, as well as continue
learning novel methods in mathematical modeling and machine-learning.

## Education

`2014 - 2019`
__University of Edinburgh__
Ph.D. in Cell and Molecular Biology.  <br />
Thesis title: "Growth influences the single cell variability of the DNA damage response in Escherichia coli"

`2012 - 2014`
__Universite Paris Descartes__
M.Sc. in Life Sciences

`2006 - 2009`
__Universidad de Chile__
B.Sc. in Biological Sciences

## Experience

`Oct 2014 - Mar 2019`
__PhD in Cell and Molecular Biology__ 
University of Edinburgh
- Studied the response of bacteria to DNA damage from antibiotics at the single cell level using microscopy and microfluidics techniques.
- Optimised and automated the acquisition of images, improving the output by one order of magnitude.
- Developed an algorithm for automated image segmentation, and a graphical user interface for the analysis and inspection of large datasets (more than 10 GB).
- Developed a dynamical model and inferred relevant biological parameters from my data, which lead to a publication describing novel relations between population dynamics and the DNA damage response.
- Used next-generation sequencing to study how DNA is repaired with genome-wide resolution.
- Supervised 3 master and 2 bachelor students.

`Jun 2014 -- Nov 2014`
__Member of the Paris-Bettencourt iGEM team__ 
Universite Paris Descartes
- Participated in the International Genetically Engineered Machine (iGEM) competition, with a project devoted to tuberculosis.
- Designed the best project wiki in Europe.
- We used regression and classification to identify 10 new drug candidates from a drug screen with hundreds of compounds.
- Awarded Grand-prize in the overgrad division.


`Apr 2010 -- Apr 2012`
__Post-Bachelor Research Associate__, 
Pacific Northwest National Laboratory
- Developed and maintained metabolic pathway SQL databases.
- Developed a pipeline for combining genome annotation and predictions of gene enzymatic function into metabolic pathways SQL databases.
- Analysed proteomics datasets using a SVM algorithm.
- Provided support for the analysis microarray datasets using clustering and statistical testing.


## Publications

A list is available [online](https://scholar.google.co.uk/citations?user=taI4E9EAAAAJ)
